src/status_im/chat/events.cljs

```diff
@@ -177,12 +177,10 @@
       (if account-creation?
         {:db new-db
          :dispatch-n [event]}
-        (let [chats            (->> all-stored-chats
-                                    (map (fn [{:keys [chat-id] :as chat}]
-                                           [chat-id (assoc chat :last-message (get-last-stored-message chat-id))]))
-                                    (into {}))
-              ;;TODO temporary hide wallet chat, this code should be deleted after wallet contact will be deleted
-              chats' (dissoc chats "wallet")]
```

Comment:
we need to hide it for users who already have it, idk if it will be better to remove it from realm in migration

Reply:
and contact also https://github.com/status-im/status-react/blob/cb72190a185f83c0a39b0901ba01f1a12662e765/src/status_im/ui/screens/contacts/events.cljs#L275